modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -18,7 +18,7 @@
 
 from modin.core.dataframe.pandas.partitioning.partition import PandasDataframePartition
 from modin.core.execution.ray.common import RayWrapper
-from modin.core.execution.ray.common.utils import ObjectIDType, deserialize
```

Comment:
New utility functions can be moved to the same place where `deserialize` is defined.